codecov.yml

```diff
@@ -1,6 +1,7 @@
 ignore:
   - "env"
   - "mocks"
+  - "**/benchmarking"
```

Comment:
If api/benchmarking has good cov i guess we would like to keep that?

Reply:
so far, it has 0, will switch it not exclude this folder